### ISSUE TITLE
Plots compatibility

### ISSUE THREAD

**Issue by edwin-wang** [chiabluepool/bluepool#1]
After installed Linux build, I tried to add my plots generated by official chia-blockchain build but seems bluepool's app cannot detect them. So your app is not compatible with official plots, right?

**Comment by mendogamers** [comment]
i have the same problem with official chia-blockchain in windows

**Comment by chiabluepool** [comment]
This is correct, currently only plots created using Bluepool are supported. We are working supporting your old plots, expecting to have it ready in 2-3 days.

**Comment by chiabluepool** [comment]
I'll comment here once this is ready.

**Comment by ArtiskOnGit** [comment]
Nice!

**Comment by bigmac5753** [comment]
Looking forward to this.

**Comment by l3aiano** [comment]
Good Morning! I have two questions Friend!
1- Is Chiabluepool already farming?
2- Are you already accepting Plots created in Oficial Chia? If not, is there a forecast of when these options will be released? Many Tera-bytes here with no condition to start from scratch again get it?

**Comment by aboukouris** [comment]
> This is correct, currently only plots created using Bluepool are supported. We are working supporting your old plots, expecting to have it ready in 2-3 days.

Hello. This is very nice and would be great if you manage to archive that. 
If you manage to do that, please explain us how is possible without getting our private keys that is our biggest concern and would be happily join your pool. :)

**Comment by chiabluepool** [comment]
Old plots are supported now, but only through a win/mac installer. This version is closed source because using your own keys makes it possible to cheat the pool by submitting partials that are not winning and keeping the ones that are winning. (This is the issue HPool had before it went on a banning spree)

**Comment by chiabluepool** [comment]
> > This is correct, currently only plots created using Bluepool are supported. We are working supporting your old plots, expecting to have it ready in 2-3 days.
> 
> Hello. This is very nice and would be great if you manage to archive that.
> If you manage to do that, please explain us how is possible without getting our private keys that is our biggest concern and would be happily join your pool. :)

Sorry, It's not possible without getting a private key. You will have to add your mnemonic in order to farm previous plots.

**Comment by aboukouris** [comment]
> Old plots are supported now, but only through a win/mac installer. This version is closed source because using your own keys makes it possible to cheat the pool by submitting partials that are not winning and keeping the ones that are winning. (This is the issue HPool had before it went on a banning spree)

I see. It's a good explanation. Is it possible to enter a different wallet to receive rewards?
Also where can I found the windows installer with the version that supports old plots? Here at github I can see linux version.

**Comment by lukeomatik** [comment]
Do we have a discord server? Maybe for announcement, support, user communication

**Comment by Mikeall77** [comment]
Hi Chiabluepool, nice to meet you!
Can we use the MadMax plotter?  If connected with old plots?
